gadopt/time_stepper.py

```diff
@@ -313,15 +321,19 @@ def solve_tendency(self, i_stage, t, update_forcings=None):
             self.solution_old.assign(self.solution)
         if not self._initialized:
             raise ValueError('Time integrator {:} is not initialized'.format(self.name))
-        if update_forcings is not None:
-            update_forcings(t + self.c[i_stage]*self.dt)
+
+        if update_forcings is not None and t is not None:
+            update_forcings(t + self.c[i_stage] * self.dt)
+        elif update_forcings is not None:
```

Comment:
What's the use case here?

Reply:
This is used at the moment for level-set reinitialisation. The forcing here is the gradient calculation.

Reply:
Ok that makes sense!